pkg/serviceaccount/serviceaccount.go

```diff
@@ -0,0 +1,113 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package serviceaccount contains service accounts related logic.
+package serviceaccount
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	pgpcrypto "github.com/ProtonMail/gopenpgp/v2/crypto"
+
+	"github.com/siderolabs/go-api-signature/pkg/pgp"
+)
+
+const (
+	// SideroServiceAccountKeyEnvVar is the name of the environment variable
+	// that contains the base64-encoded service account key JSON.
+	SideroServiceAccountKeyEnvVar = "SIDERO_SERVICE_ACCOUNT_KEY"
+
+	// OmniServiceAccountKeyEnvVar is the name of the environment variable
+	// that contains the base64-encoded service account key JSON.
+	OmniServiceAccountKeyEnvVar = "OMNI_SERVICE_ACCOUNT_KEY"
```

Comment:
does it make sense to have an env var with `OMNI` prefix in `go-api-signature`?

Reply:
yes, according to @smira it's fine now